canary/build.gradle

```diff
@@ -58,14 +63,35 @@ dependencies {
         } else {
             implementation "ai.djl.tensorflow:${engine}::${os}-x86_64"
         }
+    } else if (engine.startsWith("onnxruntime")) {
+        implementation "ai.djl.mxnet:mxnet-engine"
+        implementation "ai.djl.mxnet:mxnet-native-auto"
+    } else if (engine.startsWith("paddlepaddle")) {
+        implementation "ai.djl.paddlepaddle:paddlepaddle-engine"
+        if (engine.contains("-auto")) {
+            implementation "ai.djl.paddlepaddle:paddlepaddle-native-auto"
+        } else {
+            implementation "ai.djl.paddlepaddle:${engine}::${os}-x86_64"
+        }
+        implementation "ai.djl.mxnet:mxnet-engine"
+        implementation "ai.djl.mxnet:mxnet-native-auto"
+    } else if (engine.startsWith("dlr")) {
+        implementation "ai.djl.dlr:dlr-engine"
+        if (engine.contains("-auto")) {
+            implementation "ai.djl.dlr:dlr-native-auto"
+        } else {
+            // TODO: remove version after DJL 0.10.0
+            implementation "ai.djl.dlr:${engine}:1.6.0:${os}-x86_64"
```

Comment:
why not using dlr_version

Reply:
dlr_version is not defined in gradle.properties in the this project. And once we added to bom, we don't need the dlr_version any more.